bftengine/src/bftengine/ControllerWithSimpleHistory.hpp

```diff
@@ -23,13 +23,23 @@
 namespace bftEngine {
 namespace impl {
 
+#define RELATIVE_LOWER_BOUND(rel, val) (uint32_t)((1 - rel) * val)
```

Comment:
Can we have a template method instead of macro?

Reply:
Sure

Reply:
+1